tests/comments/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -1304,6 +1306,7 @@ exports[`inline6.php 1`] = `
 /**
  * test
  */
+
```

Comment:
This should probably go, right?

Reply:
I think no, but it is not related to this PR, we insert hardline before last closing tag (if newline exists in source code), and here we have two newlines, better create new issue

Reply:
@evilebottnawi fixed it!